test/pspm_get_sps_test.m

```diff
@@ -0,0 +1,41 @@
+classdef pspm_get_sps_test < matlab.unittest.TestCase
+% SCR_GET_SPIKE_TEST 
+% unittest class for the pspm_get_sps_test function
+%__________________________________________________________________________
+% SCRalyze TestEnvironment
+% (C) 2013 Linus R�ttimann (University of Zurich)
+        
+    methods (Test)
+        function invalid_eye(this)
+            
+            import.sr = 100;
+            import.data = ones(1,1000);
+            import.units = 'degree';
+            import.range = [ 0, 1];            
```

Comment:
Maybe testing with a real data file would be better, no ?

Reply:
Yeh, I'm not sure, what's the benefit of a real data file, the fact that it's as close to the truth as possible? Makes sense although I don't have access to that kind of data, @SerenXia do you have anonymised data we could add to the testing server?

Reply:
yes, the ViSearch experiment was specifically designed for this sps method and the dataset was anonymised and uploaded in Zenodo. Or you can also use PIT1 and PIT2, both in Zenodo. However the anonymised datasets only contain data without pre-processing. I can send you a few files from these datasets if needed.

Reply:
It doesn't have to be necessarily anonymized data since they will be stored on our server on Jenkins, i.e. private. For instance the file I've just sent you would work as well ;)

Reply:
Great, I actually do have the PIT files that yanfang had sent me a while back I'll use those as even thought the server is private I think it's best to use anonymised where possible

Reply:
Hmmm, actually I'm a bit confused in this case, for the most part I just renames the get_sps function and included the l/r switch. There are other get_x function that use a datafile format and load from a file, however get_sps seems to directly get it's data from the input param import.data. @irojkov-ph do you know why this is?

Reply:
get scr is another which take data from the input arg directly, that uses generated data rather than a test file as well

Reply:
My comment in `pspm_get_sps` would change much in this test function I guess (you can test `pspm_get_sps_l` and `pspm_get_sps_r` in the same test function, no need to create two separate ones)